Sources/ProductHunt/SwiftUI/ProductHuntButton.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  ProductHuntButton.swift
+//  ProductHunt
+//
+//  Created by Julien Lacroix on 04/11/2020.
+//
+
+/// We want ProductHuntButton available for iOS, tvOS, and watchOS.
+#if canImport(UIKit)
+import SwiftUI
+import SafariServices
+
+
+
+//MARK: - View
+@available(iOS 14, *)
+public struct ProductHuntButton: View {
+    
```

Comment:
A single line break should be enough here ;)

Reply:
Done. You can check the new commit: "Clean code"